### DISCUSSION TITLE
Navigating Open Source: Strategies for New Contributors

### DISCUSSION THREAD

**Discussion by IsuruSankhajith** [community/community#113244] [Programming Help]
### Body

What are some effective strategies for contributing to open-source projects as a beginner? How can newcomers identify projects aligned with their interests and skill level, and what are some best practices for making meaningful contributions while learning and collaborating within the open-source community?

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by ghostinhershell** [comment]
Hey @IsuruSankhajith, 

Thanks so much for reaching out in our community and good question! Here are some resources that can help you get started:
- [How to choose (and contribute to) your first open source project Collection](https://github.com/collections)
- [Learn to Code Collection](https://github.com/collections/learn-to-code)
- [first-contributions](https://github.com/firstcontributions/first-contributions)

You can also find learning resources in our [GitHub Community Programs 🚀](https://github.com/orgs/community/discussions/68410) check-in post.

Happy coding! 🍀
